lib/item-main/addon/templates/components/item-meta.hbs

```diff
@@ -0,0 +1,3 @@
+<main class="shop-item__main">
+  <img src="https://via.placeholder.com/160" alt={{className}}>
```

Comment:
We probably want to not use `className` since this isnt actually a class name. Rather we should pass in `imgAlt` or something like that.

Can we also refactor the data returned by the route so it doesnt use `className`?

Reply:
ya good point